api/graphql/tests/test_reservations/test_reservation_create.py

```diff
@@ -499,14 +500,19 @@ def test_create_fails_when_reservation_unit_min_reservation_duration_subsides(se
 
     def test_create_succeeds_when_start_time_matches_reservation_start_interval(self):
         self.client.force_login(self.regular_joe)
-        intervals = list(ReservationStartInterval.values)
-        for interval, interval_minutes in zip(intervals, [15, 30, 60, 90]):
+
+        interval: ReservationStartInterval
+        for interval in ReservationStartInterval:
```

Comment:
If you have the time, these should be rewritten as parametrized functional tests.

Reply:
I will wait until #1115 is merged and then I can rewrite all of the tests in this file without worry of creating conflicts for you :)